### PR TITLE
Cleanup docs and standardize structs

### PR DESCRIPTION
**Pull request by MarshallOfSound**
While parsing the docs into typescript I throw a few errors that are slightly more strict than the docs linter.  These fix a few of the issues 👍 

/cc @zeke

### CODE REVIEW

File: docs/api/structures/cookie.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Certificate Object
+
+*  `name` String - The name of the cookie.
```
Comment (zeke) [inline docs/api/structures/cookie.md]: these bullets are all followed by double spaces 🙌

Comment (MarshallOfSound) [inline docs/api/structures/cookie.md]: What happened here 😢 I shall fix 😆

File: docs/api/structures/bluetooth-device.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Bluetooth Device Object
```
Comment (zeke) [inline docs/api/structures/bluetooth-device.md]: BluetoothDevice

**Comment by kevinsawicki** [comment]
👍 📝

**Pull request merged**